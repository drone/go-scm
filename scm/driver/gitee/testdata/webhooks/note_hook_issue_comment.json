{
  "action": "comment",
  "author": {
    "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
    "email": "qkssk1711@163.com",
    "html_url": "https://gitee.com/kit101",
    "id": 1535738,
    "login": "kit101",
    "name": "kit101",
    "remark": null,
    "site_admin": false,
    "type": "User",
    "url": "https://gitee.com/kit101",
    "user_name": "kit101",
    "username": "kit101"
  },
  "comment": {
    "body": "test issue comment hook\r\n\r\n",
    "created_at": "2021-10-08T15:45:08+08:00",
    "html_url": "https://gitee.com/kit101/drone-yml-test/issues/I4CF12#note_6933558",
    "id": 6933558,
    "updated_at": "2021-10-08T15:45:08+08:00",
    "user": {
      "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
      "email": "qkssk1711@163.com",
      "html_url": "https://gitee.com/kit101",
      "id": 1535738,
      "login": "kit101",
      "name": "kit101",
      "remark": null,
      "site_admin": false,
      "type": "User",
      "url": "https://gitee.com/kit101",
      "user_name": "kit101",
      "username": "kit101"
    }
  },
  "enterprise": null,
  "hook_id": 788005,
  "hook_name": "note_hooks",
  "hook_url": "https://gitee.com/kit101/drone-yml-test/hooks/788005/edit",
  "issue": {
    "assignee": {
      "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
      "email": "qkssk1711@163.com",
      "html_url": "https://gitee.com/kit101",
      "id": 1535738,
      "login": "kit101",
      "name": "kit101",
      "remark": null,
      "site_admin": false,
      "type": "User",
      "url": "https://gitee.com/kit101",
      "user_name": "kit101",
      "username": "kit101"
    },
    "body": "create by api body",
    "collaborators": [
    ],
    "comments": 2,
    "created_at": "2021-09-29T13:33:00+08:00",
    "html_url": "https://gitee.com/kit101/drone-yml-test/issues/I4CF12",
    "id": 7297814,
    "labels": [
      {
        "color": "DB2828",
        "id": 102145590,
        "name": "bug"
      }
    ],
    "milestone": null,
    "number": "I4CF12",
    "state": "open",
    "state_name": "待办的",
    "title": "create by api",
    "type_name": "任务",
    "updated_at": "2021-10-08T15:45:08+08:00",
    "user": {
      "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
      "email": "qkssk1711@163.com",
      "html_url": "https://gitee.com/kit101",
      "id": 1535738,
      "login": "kit101",
      "name": "kit101",
      "remark": null,
      "site_admin": false,
      "type": "User",
      "url": "https://gitee.com/kit101",
      "user_name": "kit101",
      "username": "kit101"
    }
  },
  "note": "test issue comment hook\r\n\r\n",
  "noteable_id": 7297814,
  "noteable_type": "Issue",
  "password": "",
  "per_iid": "#I4CF12",
  "project": {
    "clone_url": "https://gitee.com/kit101/drone-yml-test.git",
    "created_at": "2021-03-24T11:24:34+08:00",
    "default_branch": "master",
    "description": "",
    "fork": false,
    "forks_count": 0,
    "full_name": "kit101/drone-yml-test",
    "git_http_url": "https://gitee.com/kit101/drone-yml-test.git",
    "git_ssh_url": "git@gitee.com:kit101/drone-yml-test.git",
    "git_svn_url": "svn://gitee.com/kit101/drone-yml-test",
    "git_url": "git://gitee.com/kit101/drone-yml-test.git",
    "has_issues": true,
    "has_pages": false,
    "has_wiki": true,
    "homepage": "https://gitee.com/kit101/drone-yml-test",
    "html_url": "https://gitee.com/kit101/drone-yml-test",
    "id": 14836026,
    "language": null,
    "license": null,
    "name": "drone-yml-test",
    "name_with_namespace": "kit101/drone-yml-test",
    "namespace": "kit101",
    "open_issues_count": 2,
    "owner": {
      "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
      "email": "qkssk1711@163.com",
      "html_url": "https://gitee.com/kit101",
      "id": 1535738,
      "login": "kit101",
      "name": "kit101",
      "remark": null,
      "site_admin": false,
      "type": "User",
      "url": "https://gitee.com/kit101",
      "user_name": "kit101",
      "username": "kit101"
    },
    "path": "drone-yml-test",
    "path_with_namespace": "kit101/drone-yml-test",
    "private": false,
    "pushed_at": "2021-10-08T13:26:59+08:00",
    "ssh_url": "git@gitee.com:kit101/drone-yml-test.git",
    "stargazers_count": 0,
    "svn_url": "svn://gitee.com/kit101/drone-yml-test",
    "updated_at": "2021-10-08T13:26:59+08:00",
    "url": "https://gitee.com/kit101/drone-yml-test",
    "watchers_count": 1
  },
  "pull_request": null,
  "push_data": null,
  "repository": {
    "clone_url": "https://gitee.com/kit101/drone-yml-test.git",
    "created_at": "2021-03-24T11:24:34+08:00",
    "default_branch": "master",
    "description": "",
    "fork": false,
    "forks_count": 0,
    "full_name": "kit101/drone-yml-test",
    "git_http_url": "https://gitee.com/kit101/drone-yml-test.git",
    "git_ssh_url": "git@gitee.com:kit101/drone-yml-test.git",
    "git_svn_url": "svn://gitee.com/kit101/drone-yml-test",
    "git_url": "git://gitee.com/kit101/drone-yml-test.git",
    "has_issues": true,
    "has_pages": false,
    "has_wiki": true,
    "homepage": "https://gitee.com/kit101/drone-yml-test",
    "html_url": "https://gitee.com/kit101/drone-yml-test",
    "id": 14836026,
    "language": null,
    "license": null,
    "name": "drone-yml-test",
    "name_with_namespace": "kit101/drone-yml-test",
    "namespace": "kit101",
    "open_issues_count": 2,
    "owner": {
      "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
      "email": "qkssk1711@163.com",
      "html_url": "https://gitee.com/kit101",
      "id": 1535738,
      "login": "kit101",
      "name": "kit101",
      "remark": null,
      "site_admin": false,
      "type": "User",
      "url": "https://gitee.com/kit101",
      "user_name": "kit101",
      "username": "kit101"
    },
    "path": "drone-yml-test",
    "path_with_namespace": "kit101/drone-yml-test",
    "private": false,
    "pushed_at": "2021-10-08T13:26:59+08:00",
    "ssh_url": "git@gitee.com:kit101/drone-yml-test.git",
    "stargazers_count": 0,
    "svn_url": "svn://gitee.com/kit101/drone-yml-test",
    "updated_at": "2021-10-08T13:26:59+08:00",
    "url": "https://gitee.com/kit101/drone-yml-test",
    "watchers_count": 1
  },
  "sender": {
    "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
    "email": "qkssk1711@163.com",
    "html_url": "https://gitee.com/kit101",
    "id": 1535738,
    "login": "kit101",
    "name": "kit101",
    "remark": null,
    "site_admin": false,
    "type": "User",
    "url": "https://gitee.com/kit101",
    "user_name": "kit101",
    "username": "kit101"
  },
  "short_commit_id": null,
  "sign": "Zzv7VMOnJ5SKrQhcEsL4XaC1zR1Kl1qMuxS0HbHwpBI=",
  "timestamp": "1633679110217",
  "title": "create by api",
  "url": "https://gitee.com/kit101/drone-yml-test/issues/I4CF12#note_6933558"
}