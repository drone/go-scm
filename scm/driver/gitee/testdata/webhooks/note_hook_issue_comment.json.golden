{
  "Action": "created",
  "Repo": {
    "ID": "14836026",
    "Namespace": "kit101",
    "Name": "drone-yml-test",
    "Branch": "master",
    "Private": false,
    "Clone": "https://gitee.com/kit101/drone-yml-test.git",
    "CloneSSH": "git@gitee.com:kit101/drone-yml-test.git",
    "Link": "https://gitee.com/kit101/drone-yml-test",
    "Created": "2021-03-24T11:24:34+08:00",
    "Updated": "2021-10-08T13:26:59+08:00"
  },
  "Issue": {
    "Number": 735267704950,
    "Title": "create by api",
    "Body": "create by api body",
    "Link": "https://gitee.com/kit101/drone-yml-test/issues/I4CF12",
    "Labels": [
      "bug"
    ],
    "Closed": false,
    "Author": {
      "Login": "kit101",
      "Avatar": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png"
    },
    "Created": "2021-09-29T13:33:00+08:00",
    "Updated": "2021-10-08T15:45:08+08:00"
  },
  "Comment": {
    "ID": 6933558,
    "Body": "test issue comment hook\r\n\r\n",
    "Author": {
      "Avatar": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
      "Email": "qkssk1711@163.com",
      "Login": "kit101",
      "Name": "kit101"
    },
    "Created": "2021-10-08T15:45:08+08:00",
    "Updated": "2021-10-08T15:45:08+08:00"
  },
  "Sender": {
    "Avatar": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
    "Email": "qkssk1711@163.com",
    "Login": "kit101",
    "Name": "kit101"
  }
}
